setup.py

```diff
@@ -146,9 +146,12 @@ def build_cmake(self, ext):
             cmake_args += [
                 "-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=" + str(lib_output_dir),
             ]
+            cpu_count = multiprocessing.cpu_count()
+            if cpu_count > 2:
```

Comment:
>=?
```suggestion
            if cpu_count >= 2:
```

Reply:
intentional, but I might just add a new arg to override this behaviour